lib/perl/Genome/VariantReporting/Framework/Component/Base.pm

```diff
@@ -50,7 +50,16 @@ sub properties_in_plan {
             implied_by => undef,
             is_structural => undef)) {
         next if $property->{is_param} && $property->property_name =~ m/_md5$/;
-        next if $property->{is_param} && $property->property_name =~ m/_count$/;
+        if ($property->{is_param} && $property->property_name =~ m/^(.*)_count$/) {
```

Comment:
We thought it was unlikely that an expert would have a property that ends in _md5 so we didn't think it was necessary. I can add it in for completeness if you'd like.